### PR TITLE
Implement Webhook Connection

### PR DESCRIPTION
**Pull request by ShoboySnr**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var gulp = require('gulp');
```
Comment (w3guy) [inline gulpfile.js:1]: Why did you add this file?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (w3guy) [inline package.json:1]: Why did you add this file?

File: src/Admin/Customizer/CustomControls/WP_Customize_Integration_Repeater_Control.php [hunk 3]
```diff
@@ -609,7 +655,7 @@ public function render_content()
         Apparently, it is automatically inserted by customizer*/
         ?>
         <div class="mo-integration__add_new">
-            <button type="button" class="button mo-add-new-integration">
+            <button type="button" class="button mo-integration-widget-actions">
```
Comment (w3guy) [inline src/Admin/Customizer/CustomControls/WP_Customize_Integration_Repeater_Control.php:658]: Why was the class name changed?

File: src/assets/images/admin-icon.svg [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!-- Creator: CorelDRAW 2018 (64-Bit) -->
```
Comment (w3guy) [inline src/assets/images/admin-icon.svg:3]: Why is this file here when it has been removed?

**Review by w3guy — CHANGES_REQUESTED** [review]
Clean up the PR and answer my questions so I can work on the data persistence.

CC @ShoboySnr

File: src/Admin/Customizer/CustomControls/WP_Customize_Integration_Repeater_Control.php (hunk 3 above)
Comment (w3guy) [reply src/Admin/Customizer/CustomControls/WP_Customize_Integration_Repeater_Control.php:658]: @ShoboySnr did yoy miss this?

File: src/assets/images/admin-icon.svg (hunk 4 above)
Comment (w3guy) [reply src/assets/images/admin-icon.svg:3]: @ShoboySnr did you miss this comment?

File: src/assets/js/customizer-controls/integration-control/style.css [hunk 5]
```diff
@@ -211,4 +211,8 @@ a.mo-integration-form-remove, a.mo-integration-delete {
 
 .customize-control textarea {
     width: 100%;
+}
+
+a.mo-webhook-header-fields-remove {
```
Comment (w3guy) [inline src/assets/js/customizer-controls/integration-control/style.css:216]: @ShoboySnr is this css used anywhere?

**Review by w3guy — CHANGES_REQUESTED** [review]
See my comments

File: src/assets/js/customizer-controls/integration-control/style.css (hunk 5 above)
Comment (ShoboySnr) [reply src/assets/js/customizer-controls/integration-control/style.css:216]: @w3guy yes it is. the button that removes the fields row when appended

File: src/Admin/Customizer/CustomControls/WP_Customize_Integration_Repeater_Control.php (hunk 3 above)
Comment (ShoboySnr) [reply src/Admin/Customizer/CustomControls/WP_Customize_Integration_Repeater_Control.php:658]: I added a different custom css styling to this.
